enterprise/backend/test/metabase_enterprise/advanced_permissions/query_processor/middleware/permissions_test.clj

```diff
@@ -80,8 +80,8 @@
         (testing "Native queries are unmodified"
           (is (= (native-download-query) (ee.qp.perms/apply-download-limit (native-download-query))))))
 
-      (with-download-perms! (mt/id) {:schemas {"PUBLIC" {(mt/id 'venues) :limited
-                                                         (mt/id 'checkins) :full}}}
```

Comment:
Changed all these since keywords are used in `mt/id` everywhere else in the codebase